autoscheduler/frontend/src/components/GoogleButton.tsx

```diff
@@ -0,0 +1,25 @@
+// import * as React from 'react';
+// import * as ReactDOM from 'react-dom';
+// import GoogleLogin from 'react-google-login';
+//
+//
+// const responseGoogle = () => {
+//   console.log(Response);
+// }
+//
+// const GoogleLoginButton: React.SFC = function App() {
+//   return (
+// ReactDOM.render(
+//   <GoogleLogin
+//     clientId="658977310896-knrl3gka66fldh83dao2rhgbblmd4un9.apps.googleusercontent.com"
+//     buttonText="Login"
+//     onSuccess={responseGoogle}
+//     onFailure={responseGoogle}
+//     cookiePolicy={'single_host_origin'}
+//   />,
+//   document.getElementById('googleButton')
+//
+// );
+// }
+//
+// export default GoogleLoginButton;
```

Comment:
Are we eventually going to use this? If so, remake the file so it's more of an outline of what it should be(rather than having specific with the GoogleLogin)

Reply:
I was tinkering with authenticating with the google login button, but didn't get that far obviously. Didn't mean to commit this file, I must've accidentally tracked it

Reply:
If you'd like to, you can undo the most recent commit by doing `git reset HEAD~`, then remove this from being tracked and remake the commit like normal, then do `git push --force`.

Reply:
Actually nvm, for the sake of keeping the PR readable just make it in a separate commit removing the file, then before we merge we can squash the commit with the first/current one.